docs/clustering/point/index.md

```diff
@@ -0,0 +1,28 @@
+# `multiple_object_tracking::Point`
+
+Defined in header `<multiple_object_tracking/clustering.hpp>`
+
+```cpp
+struct Point;
+```
+
+This class represents the geometric center of a cluster. It is a sub-vector of `multiple_object_tracking::CtrvState`
```

Comment:
Technically not just the geometric center, may want to clarify that the centroid uses all the members of Point as it is not solely positional